client/src/cmdhfmf.c

```diff
@@ -5167,9 +5182,9 @@ static int CmdHF14AMfMAD(const char *Cmd) {
     CLIParserContext *ctx;
     CLIParserInit(&ctx, "hf mf mad",
                   "Checks and prints MIFARE Application Directory (MAD)",
-                  "hf mf mad -> shows MAD if exists\n"
-                  "hf mf mad --aid e103 -k ffffffffffff -b -> shows NDEF data if exists. read card with custom key and key B\n"
-                  "hf mf mad --dch -k ffffffffffff -> decode CardHolder information\n");
+                  "hf mf mad        			--> Shows MAD if it exists\n" //4 spaces, 3 tabs
+                  "hf mf mad --dch -k ffffffffffff 	     --> Decode CardHolder information\n" //6 spaces, 1 tab
+                  "hf mf mad --aid e103 -k ffffffffffff -b  --> Shows NDEF data if it exists. Read card with custom key and key B\n"); //2 spaces
```

Comment:
remove the confusing end line comments.  and try line up the `-->`  so it will end up same in the client.